drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -307,34 +307,70 @@
 
 
 /* Font Awesome icons have different width - margin-right tries to compensate it */
-.xblock--drag-and-drop .feedback p:before {
+.ltr .xblock--drag-and-drop .feedback p:before,
```

Comment:
@alisan617 or @bjacobel, is there a better way to handle rtl/ltr? I don't recall seeing this pattern anyplace else.